tests/integrations/config/test-custom.toml

```diff
@@ -618,7 +618,6 @@ recovery-mode = "tolerate-tail-corruption"
 recovery-read-block-size = "1KB"
 recovery-threads = 2
 memory-limit = "1GB"
-enable-log-recycle = true # enable by default
```

Comment:
This is supposed to test configs that are different from default. "enable-log-recycle = false"